spec/tests/check.fmf

```diff
@@ -14,6 +14,27 @@ description: |
     panic detection, core dump collection or collection of system
     logs.
 
+    By default, the check results affect the overall test outcome.
+    To change this behaviour, use the ``result`` key, which accepts
+    the following values:
+
+        respect
+            The check result is respected and affects the overall
+            test result. This is the default.
+
+        xfail
+            The check result is expected to fail (pass becomes
+            fail and vice-versa).
+
+        info
+            The check result is treated as an informational
+            message and does not affect the overall test result.
+
+    Note that running one check multiple times for the same test is
```

Comment:
@psss Could we use make this more visible via a warning or note pls?

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Reply:
Right, we discussed this yesterday @psss - would it be hard to add the detection and warning during discovery?

Reply:
Ack, fixed in cd0527d4.